src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -82,23 +82,54 @@ export class NetworkProcessor {
       NetworkProcessor.parseUrlString(params.url);
     }
 
-    const {url, method} = params;
+    const {url, method, headers} = params;
     // TODO: Set / expand.
-    // ; headers
     // ; cookies
     // ; body
 
+    const requestHeaders: Protocol.Fetch.HeaderEntry[] | undefined =
+      cdpFetchHeadersFromBidiNetworkHeaders(headers);
+
     await this.#networkStorage
       .getRequest(networkId)
-      ?.continueRequest(fetchId, url, method);
+      ?.continueRequest(fetchId, url, method, requestHeaders);
 
     this.#networkStorage.removeBlockedRequest(networkId);
 
     return {};
   }
 
-  continueResponse(_params: Network.ContinueResponseParameters): EmptyResult {
-    throw new UnknownCommandException('Not implemented yet.');
+  async continueResponse(
+    params: Network.ContinueResponseParameters
+  ): Promise<EmptyResult> {
+    const networkId = params.request;
+    const {request: fetchId, phase} = this.#getBlockedRequest(networkId);
+
+    if (phase === Network.InterceptPhase.BeforeRequestSent) {
+      throw new InvalidArgumentException(
+        `Blocked request for network id '${networkId}' is in 'BeforeRequestSent' phase`
+      );
+    }
+
+    const {statusCode, reasonPhrase, headers} = params;
+
+    // TODO: Step 6
+    // https://w3c.github.io/webdriver-bidi/#command-network-continueResponse
+
+    const responseHeaders: Protocol.Fetch.HeaderEntry[] | undefined =
+      cdpFetchHeadersFromBidiNetworkHeaders(headers);
+
+    // TODO: Step 7.
+    // TODO: Set / expand.
+    // ; cookies
+    // ; credentials
+    await this.#networkStorage
+      .getRequest(networkId)
+      ?.continueResponse(fetchId, statusCode, reasonPhrase, responseHeaders);
```

Comment:
You already have a request from `getBlockedRequest`. Why getting it again?

Reply:
Not quite. `getBlockedRequest` does not return a `NetworkRequest` object. We need to get it directly from the corresponding map, with `getRequest`.

I do agree about the `?.` though, the request should be present at this point, so in theory we could replace it with `!.`. Or even add an assertion. I'm generally fonder of using `?.` just as a matter of defensive programming.

Thoughts?

Reply:
Assertion at the beginning of the method SGTM

Reply:
Done